solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -82,6 +82,9 @@ library DKG {
         // Identifiers of candidate group members as outputted by the group
         // selection protocol.
         uint32[] members;
```

Comment:
I'm sorry I missed it before, shouldn't this be renamed to `selectedOperators`?

Reply:
Actually, this may require a broader naming cleanup. So we assume that operators are selected from the sortition pool to participate in a DKG. Once operators complete the DKG successfully they become group members. 

I propose the following naming convention:

|old name|new name|
|:---:|:---:|
|`members`|`selectedOperators`|
|`submitterMemberIndex`|`submitterOperatorIndex`|
|`misbehavedMembersIndices`|`misbehavedOperatorsIndices`|
|`signingMembersIndices`|`signingOperatorsIndices`|
|`membersHash`|`groupMembersHash`|

Refs our previous conversation https://github.com/keep-network/keep-core/pull/2768#discussion_r773098164

Reply:
CC: @pdyraga @dimpar

Reply:
> Once operators complete the DKG successfully they become group members.

What do you consider a successful DKG completion? Is it upon dkg submission or when the challenge period is passed and dkg is approved?

Reply:
> > Once operators complete the DKG successfully they become group members.
> 
> What do you consider a successful DKG completion? Is it upon dkg submission or when the challenge period is passed and dkg is approved?

submitDkgResult + approveDkgResult